plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/fake_quantize.cpp

```diff
@@ -0,0 +1,116 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "common_test_utils/test_constants.hpp"
+#include "single_layer_tests/fake_quantize.hpp"
+
+using namespace LayerTestsDefinitions;
+
+namespace {
+
+const std::vector<InferenceEngine::Precision> netPrecisions = {
+    InferenceEngine::Precision::FP32,
+    // InferenceEngine::Precision::FP16,
+};
+
+const std::vector<std::vector<size_t>> inputShapes = {
+    {1, 1},           {2, 6},           {1, 1, 1},         {2, 6, 13},       {1, 1, 1, 1},
+    {3, 10, 5, 6},    {2, 8, 5, 18},    {2, 16, 3, 18},    {3, 49, 5, 6},    {1, 1, 1, 1, 1},
+    {3, 10, 2, 5, 6}, {2, 8, 1, 5, 18}, {2, 16, 4, 3, 18}, {3, 49, 7, 5, 6},
```

Comment:
If adding `,` at the end still not making one element per line, please add `//` after each element.

Reply:
OK, as it seems openvino has different formating style, will add `//` to make them aligned.

Reply:
Thanks, Tim mentioned he would prefer this format. So let's keep using the one element per line format.

Reply:
Done.

Reply:
Thanks. Please submit this to plaidml.